$NetBSD: patch-af,v 1.1 2008/02/17 03:30:10 bsadewitz Exp $

Turn off transparency by default--those who need it can turn it on,
and it doesn't make sense for non-xterm use.

--- src/config/options.inc.orig	2008-02-15 18:40:03.000000000 -0500
+++ src/config/options.inc
@@ -896,7 +896,7 @@ static struct option_info config_options
 		"4 is true color mode, uses konsole RGB codes")),
 
 	INIT_OPT_BOOL("terminal._template_", N_("Transparency"),
-		"transparency", 0, 0,
+		"transparency", 0, 1,
 		N_("If we should not set the background to black. This is particularly\n"
 		"useful when we have a terminal (typically in some windowing\n"
 		"environment) with a background image or a transparent background -\n"
